### PR TITLE
Feature/issue994

### PR DESCRIPTION
**Pull request by chrisala**

### CODE REVIEW

File: grails-app/conf/data/mapping.json [hunk 1]
```diff
@@ -70,17 +70,16 @@
...
-          "copy_to": ["organisationId"]
-        },
         "organisationName": {
           "type" : "text",
           "copy_to": ["organisationFacet", "organisationSort"]
         },
         "organisationFacet": {
           "type": "keyword"
         },
+        "linkedOrganisationFacet": {
```
Comment (temi) [inline grails-app/conf/data/mapping.json:80]: is this required?

**Review by temi — COMMENTED** [review]
A few questions to double check my understanding.

File: scripts/releases/5.0/updateAssociatedOrgNames.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+load('../../utils/audit.js');
+let projects = db.project.find({status:{$ne:'deleted'}, associatedOrgs:{$exists:true}, isMERIT:false});
```
[the remark this answers is not in the document]
Comment (chrisala) [reply scripts/releases/5.0/updateAssociatedOrgNames.js:2]: Yes, my concern is that I've changed toMap to no longer do the biocollect override of the associatedOrg.name with the organisation name if the associatedOrg has an organisationId.  A while back we discussed updating the data so BioCollect will still retain this behaviour even after this change is deployed.

File: grails-app/conf/data/mapping.json (hunk 1 above)
Comment (chrisala) [reply grails-app/conf/data/mapping.json]: I've actually just got the requirements for how the organisation facet should work in MERIT this morning - we were originally going to have 2 organisation facets with slightly different behaviour (which is what this was going to be used for) but I think we no longer need it so I've removed it.

File: scripts/releases/5.0/updateAssociatedOrgNames.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (temi) [reply scripts/releases/5.0/updateAssociatedOrgNames.js:2]: yes, this will need to be tested on BioCollect.

**Review by temi — APPROVED** [review]

**Pull request merged**